src/apis/software-service.ts

```diff
@@ -0,0 +1,109 @@
+import { DocumentClient } from 'aws-sdk/clients/dynamodb';
+import { v4 as uuidv4 } from 'uuid';
+import { SoftwareModel, Status } from './schemas/software-registry/software';
```

Comment:
It's a relatively minor thing but use double quotes everywhere throughout the code to keep consistency with the rest of the project